### ISSUE TITLE
./configure fails with weird errors when `which` is not installed

### ISSUE THREAD

**Issue by AtnNn** [rethinkdb/rethinkdb#1572]
For example, I sometimes get this error:

```
C++ Compiler:                   error
* Error: unable to run /usr/bin/g++
```

Installing which fixes the problem.

**Comment by AtnNn** [comment]
I replaced `which` with `command -v` which should be more portable. In review assigned to @larkost.

**Comment by AtnNn** [comment]
I forgot to close this issue. The fix, c7597ef, is in next and v2.0.x
